tests/system/test_system_dbapi.py

```diff
@@ -39,6 +39,9 @@
 )
 
 
+SPANNER_RPC_TIMEOUT_IN_SECONDS = int(os.getenv("SPANNER_RPC_TIMEOUT_IN_SECONDS", 30))
```

Comment:
Please update this to be 60 seconds as well 👍

Reply:
Updated the name and value. Thanks!